boards/arm/bbc_microbit/bbc_microbit.dts

```diff
@@ -17,6 +17,7 @@
 		sw0 = &buttonA;
 		sw1 = &buttonB;
 		magn0 = &lsm303agr_magn;
+		accel0 = &mma8653fc;
 		watchdog0 = &wdt0;
```

Comment:
accel1?

Reply:
There are some variations of the BBC microbit. Each has one accelerometer, but the type of sensor is different (LSM303AGR or MMA8653FC).

The dts enables MMA8653FC sensor, and other sensors are disabled.
So, I set mma8653fc to accel0 and no use lsm303agr.

(I think it is intend to enable/disable each sensors by overlay file.)

Reply:
There are some variations of the BBC microbit. Each has one accelerometer, but the type of sensor is different (LSM303AGR or MMA8653FC).

The dts enables MMA8653FC sensor, and other sensors are disabled.
So, I set mma8653fc to accel0 and no use lsm303agr.

(I think it is intend to enable/disable each sensors by overlay file.)